src/objects/zcl_abapgit_object_tabl.clas.abap

```diff
@@ -170,13 +168,9 @@ CLASS ZCL_ABAPGIT_OBJECT_TABL IMPLEMENTATION.
     DATA lv_segment_type        TYPE edilsegtyp.
     DATA lv_result              LIKE sy-subrc.
 
-    TRY.
-        check_is_idoc_segment( ).
-
-      CATCH zcx_abapgit_exception ##no_handler.
-        rv_deleted = abap_false.
-        RETURN. "previous XML version or no IDoc segment
-    ENDTRY.
+    IF is_idoc_segment( ) = abap_false.
+      RETURN. "previous XML version or no IDoc segment
```

Comment:
set `rv_deleted = abap_false` like the old code?

Reply:
Sure